### ISSUE TITLE
Directory not in repo: ./local

### ISSUE THREAD

**Issue by harding** [stickies-v/vs-core#1]
From the README:

```
git clone https://github.com/stickies-v/vs-core
cd vs-core/local/
devcontainer open
```

There is no local/ directory.

**Comment by stickies-v** [comment]
Ah yes, the `local` subdir is an artefact from earlier code organization. Will update, thanks.

**Comment by stickies-v** [comment]
Fixed in 7288f4c767c9764d448c4b423a4c8c76ddae97d8.
